molo/core/tests/test_views.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+import pytest
+
+
+@pytest.mark.django_db
+class TestPages(TestCase):
+    fixtures = ['molo/core/tests/fixtures/test.json']
+
+    def test_breadcrumbs(self):
+        response = self.client.get('/')
+        assert response.status_code == 200
+        assert 'Home' not in response.content
```

Comment:
Rather use the existing `assertContains` helpers, that takes care of things like encoding.